index.html

```diff
@@ -606,25 +606,9 @@ <h4>
           <p>
             Posture value readings are explicitly flagged by the Secure
             Contexts specification as a high-value target for network
-            attackers. Thus all interfaces defined by this specification or
-            extension specifications are only available within a secure
-            context.
-          </p>
-        </section>
-        <section>
-          <h4>
-            Focused Area
-          </h4>
-          <p>
-            Posture value readings are only available for each
-            [=navigable/active document=] whose origin is the same
-            origin-domain with the currently focused area {{Document}}.
-          </p>
-          <p>
-            This is done in order to mitigate the risk of a skimming attack
-            against the browsing context containing an element which has gained
-            focus, for example when the user carries out an in-game purchase
-            using a third party payment service from within an iframe.
+            attackers. {{DevicePosture}} has the {{SecureContext}}
+            extended attribute in its declaration, and so does the
+            addition to the {{Navigator}} interface.
```

Comment:
```suggestion
            addition to the {{Navigator}} interface (`[[[##extensions-to-the-navigator-interface]]]`.
```

Otherwise there's just going to be a link to the Navigator interface in the HTML spec.